client/tasks/fills/appearance.js

```diff
@@ -137,8 +137,14 @@ class Appearance extends Component {
 				this.setState( { isPending: false } );
 				this.completeStep();
 			} )
-			.catch( ( error ) => {
-				createNotice( 'error', error.message );
+			.catch( () => {
+				createNotice(
+					'error',
+					__(
+						'There was an error importing the sample products',
```

Comment:
By removing the `error.message` we remove the ability to get our messages sent from the server, which we currently do use:

https://github.com/woocommerce/woocommerce-admin/blob/4f44222b3d10eeacaeeb50d166e73005a1149e22/src/API/OnboardingTasks.php#L318

We could wrap the `import_sample_products` in a try/catch, but something tells me this error came from JN and I think this can happen anywhere during the REST lifecycle, so that would not fully resolve the issue